resources/assets/js/components/Select.tsx

```diff
@@ -66,7 +66,7 @@ const Select: React.FunctionComponent<SelectProps> = ({
       <select
         id={id}
         name={name}
-        value={selected || ""}
+        value={selected !== null ? selected : ""}
```

Comment:
Curious as to why you changed this? More explicit?

Reply:
Because it meant you couldn't use any falsey values as options, including 0 (zero). When it explictly checks for null, 0 becomes a valid option.